src/components/Header/Header.tsx

```diff
@@ -1,22 +1,22 @@
-import { FC } from 'react'
-import AppBar from '@mui/material/AppBar'
+import { FC } from 'react';
+import AppBar from '@mui/material/AppBar';
 import { useTheme } from '@mui/material/styles';
 import useMediaQuery from '@mui/material/useMediaQuery';
 import MobileHeader from './MobileHeader';
 import DesktopHeader from './DesktopHeader';
 
-
-
 const Header: FC = () => {
-    const theme = useTheme()
```

Comment:
Not really sure why we are rewriting this. Prefer the original way

Reply:
Sorry, my formatter adds semicolons :orz: will remove these

Reply:
Here I'm referring to the refactor from lines 9-20

Reply:
The test coverage could not reach `inner = (<MobileHeader />)` for some reason so I rewrote it but let me check it again